protocol/bin/flow.js

```diff
@@ -46,31 +49,74 @@ function load (file, project) {
 }
 
 function analyze (json, project) {
+  return reduceArray([].concat(analyzeTypes(json, project), analyzeMessages(json, project), analyzeEnums(json, project)))
+}
+
+function fixCase (s) {
+  return s.toLowerCase().replace(/(_\w)/g, s => capitalize(s[1]))
+}
+
+function analyzeEnums (json, project) {
+  return json.types.filter(t => t.type === 'enum').map(t => {
+    var en = {}
+
+    t.symbols.forEach(function (s) {
+      const parts = s.split('_')
+      const val = parseInt(parts.pop(), 10)
+      const name = fixCase(parts.join('_'))
+      en[name] = val
+    })
+
+    return {
+      name: `${capitalize(json.protocol)}${t.name}`,
+      map: en,
+    }
+  }).reduce((acc, t) => {
+    return acc.concat([
+      `export const ${t.name} = {
```

Comment:
:sheep: `dedent` is pretty nice for this https://www.npmjs.com/package/dedent-js